### PR TITLE
Updates to Polymer 3.x for latest type generator changes

### PR DESCRIPTION
**Pull request by aomarks**
See https://github.com/Polymer/tools/pull/599 for context.

I'll send a follow up CL once the new type generator is published to actually begin generating and publishing the 3.0 types.

### CODE REVIEW

File: lib/elements/dom-repeat.js [hunk 1]
```diff
@@ -9,19 +9,18 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 */
 import { PolymerElement } from '../../polymer-element.js';
 
-import { TemplateInstanceBase as TemplateInstanceBase$0, templatize, modelForElement as modelForElement$0 } from '../utils/templatize.js';
+import { TemplateInstanceBase, templatize, modelForElement as modelForElement$0 } from '../utils/templatize.js';
```
Comment (azakus) [inline lib/elements/dom-repeat.js]: Add `// eslint-disable-line no-unused-vars` at the end

File: lib/legacy/templatizer-behavior.js [hunk 2]
```diff
@@ -7,9 +7,7 @@ The complete set of contributors may be found at http://polymer.github.io/CONTRI
 Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 */
-import { TemplateInstanceBase as TemplateInstanceBase$0, templatize as templatize$0, modelForElement as modelForElement$0 } from '../utils/templatize.js';
-
-let TemplateInstanceBase = TemplateInstanceBase$0; // eslint-disable-line
+import { TemplateInstanceBase, templatize as templatize$0, modelForElement as modelForElement$0 } from '../utils/templatize.js';
```
Comment (azakus) [inline lib/legacy/templatizer-behavior.js]: Add `// eslint-disable-line no-unused vars` at the end

File: lib/elements/dom-repeat.js (hunk 1 above)
Comment (aomarks) [reply lib/elements/dom-repeat.js]: Done.

File: lib/legacy/templatizer-behavior.js (hunk 2 above)
Comment (aomarks) [reply lib/legacy/templatizer-behavior.js]: Done.

**Comment by aomarks** [comment]
ping @azakus

**Comment by aomarks** [comment]
> LGTM Can we maybe include the new types already since you issued a new update on the generator?

Still need to get that actually released, will do another PR.

**Pull request merged**